$OpenBSD: patch-src_server_c,v 1.1 2014/03/27 19:06:17 jasper Exp $

XXX: Don't fail to start the gnome-terminal-server when not using UTF-8.

--- src/server.c.orig	Mon Mar  3 21:48:49 2014
+++ src/server.c	Thu Mar 27 11:56:39 2014
@@ -84,11 +84,6 @@ main (int argc, char **argv)
 
   terminal_i18n_init (TRUE);
 
-  if (!g_get_charset (NULL)) {
-    g_printerr ("Non UTF-8 locale is not supported!\n");
-    return EXIT_FAILURE;
-  }
-
 #ifndef ENABLE_DISTRO_PACKAGING
 #ifdef HAVE_UBUNTU
   /* Set some env vars to disable the ubuntu modules. Their package will be 
